### PR TITLE
Fix "make test" on mac (pkg/kubelet)

### PR DESCRIPTION
**Pull request by erictune**

### CODE REVIEW

**Comment by dchen1107** [comment]
cc/ @vishh 

Thanks for fixing the breakage. 

LGTM

File: pkg/kubelet/container_manager_unsupported_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// +build !linux
```
Comment (vishh) [inline pkg/kubelet/container_manager_unsupported_test.go]: I wonder if it is possible to share this file with both linux and non linux builds, by removing the `+build tag`? That way we can avoid duplicating this logic in the `container_manager_test.go` file.

**Comment by vishh** [comment]
Sorry about the regression. Thanks for fixing @erictune!

**Comment by vishh** [comment]
@dchen1107: I removed the LGTM for now, to see if it is possible to unify the fake implementations across build types.

**Comment by erictune** [comment]
I think fixing the build needs to take precedence over refactoring.  Please approve.

**Comment by vishh** [comment]
Sure. LGTM

File: pkg/kubelet/container_manager_unsupported_test.go (hunk 1 above)
Comment (markturansky) [inline pkg/kubelet/container_manager_unsupported_test.go]: After this merges and I rebase, I can do the refactor in https://github.com/kubernetes/kubernetes/pull/14675

There's a common fake for reuse between platforms.

**Pull request merged**